packages/synapse-react-client/src/lib/containers/widgets/facet-nav/FacetNavPanel.tsx

```diff
@@ -438,8 +438,7 @@ const FacetNavPanel: React.FunctionComponent<FacetNavPanelProps> = (
             <div className="bootstrap-4-backport SaveFiltersButtonContainer">
               <Button
                 variant="secondary"
-                className="pill-xl SaveFiltersButton"
-                size="sm"
+                className="SaveFiltersButton"
```

Comment:
This is the only instance of a small button using the `pill-xl` class -- move styles into `SaveFiltersButton`, so we can remove the `.btn.pill-xl.btn-sm` styling from `_base-import.scss`. Update border-radius and text-transform, but otherwise maintain styling of button:  

main | feature 
:---:|:---:
<img width="624" alt="main_FacetNavPanelModal" src="https://user-images.githubusercontent.com/26949006/231905156-95e86f3f-a655-428a-9f11-7e146d3d4881.png"> | <img width="627" alt="feature_FacetNavPanelModal" src="https://user-images.githubusercontent.com/26949006/231905148-d8b6480e-2dbd-466a-835e-af851b4de08c.png">